### ISSUE TITLE
Improve Accessibility

### ISSUE THREAD

**Issue by jcoona** [jcoona/Innovation-Hub#39]
In Google Chrome, there's a nice feature in Inspect Element where you can audit the webpage you are viewing, and it tells you about how efficient it is, how many stylings are being unused, etc. This would be nice to go through all the pages and make the website more effective and less spacey!

**Comment by jcoona** [comment]
CSS and JS scripts/links can be added to the end of the HTML document!

**Comment by jcoona** [comment]
Forward any http requests to be https for security.

**Comment by jcoona** [comment]
images should have an alt attribute, to either replace images with text if the image is unavailable, or just don't show the image at all `alt = ""`

**Comment by jcoona** [comment]
Input fields should have labels, so that screen readers can tell what the fields are supposed to be.

**Comment by jcoona** [comment]
add lang attribute to `html` tag.

**Comment by jcoona** [comment]
Improve color contrasts http://leaverou.github.io/contrast-ratio/#%2394969A-on-%23E9EAEB
